app/src/main/java/io/gnosis/safe/utils/BlockExplorer.kt

```diff
@@ -0,0 +1,30 @@
+package io.gnosis.safe.utils
+
+import android.content.Context
+import io.gnosis.data.models.Chain
+import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
+import pm.gnosis.model.Solidity
+import pm.gnosis.svalinn.common.utils.openUrl
+
+class BlockExplorer private constructor(
+    private val blockExplorerUrl: String
+) {
+
+    fun showAddress(context: Context, address: Solidity.Address?) {
+        //FIXME: handle possible differences in address endpoint format for different block explorers
+        context.openUrl("${blockExplorerUrl}address/${address?.asEthereumAddressChecksumString()}")
```

Comment:
is this FIXME supposed to be fixed in this PR?

Reply:
no, it is a reminder to adjust it after we get separate blockexplorer template urls from backend for transactions and addresses with placeholders